ssz/src/main/java/tech/pegasys/teku/ssz/schema/SszSchema.java

```diff
@@ -74,6 +74,16 @@ default TreeNode updateBackingNode(TreeNode srcNode, int internalIndex, SszData
     return newValue.getBackingNode();
   }
 
+  default TreeNode updateBackingNode(
+      TreeNode srcNode, int[] internalIndexes, SszData[] newValues, int count) {
+    assert internalIndexes.length >= count && newValues.length >= count;
```

Comment:
Why do we need the `count` parameter?  And shouldn't we be asserting: 
```
assert internalIndexes.length == newValues.length 
```
??

Reply:
Ugh, it was a preliminary micro-optimization I'm always arguing against :)
Replaced it with a list of structs here: dd00614 
Doesn't look that there is any performance degradation.

I'm also going to make some refactoring and move `updateBackingNode` and `createFromBackingNode(TreeNode node, int internalIndex)` to the `SszPrimitiveSchema` interface as they are relevant to packed primitives only. But this is subject for a separate PR

Reply:
Follow up refactoring: #3742